rs/types/types/src/crypto/threshold_sig/ni_dkg.rs

```diff
@@ -31,18 +32,22 @@ mod tests;
 
 /// Allows to distinguish protocol executions in high and low threshold
```

Comment:
I adapted the rustdoc in https://github.com/dfinity/ic/pull/2445/commits/643b21db78b6484c6b38921913095c8338743f61. Hopefully this makes it more clear.

Reply:
Thanks, LGTM. It may be worth it to try to more clearly differentiate between different types of keys at least in the comments like with canister threshold sigs (can we call this a canister threshold sig scheme? I think yes). Then, we can say e.g. that `LowThreshold` and `HighThreshold` is done for consensus and has just one key per subnet, while `HighThresholdForKey` is a canister threshold sig, meaning there may be many root public keys and a canister has an API to access those keys in some way, currently only VetKD.

But this is not the subject of this PR. We can also discuss this offline if this actually would make things more clear for the reader.